2020-07-03 17:46:28,750 - INFO - orm_get_article.py - get_article - 27 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:46:28,787 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:57:46,216 - INFO - orm_get_article.py - get_article - 27 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:57:46,246 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:58:44,853 - INFO - orm_get_article.py - get_article - 27 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:58:44,859 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:58:51,395 - INFO - orm_get_article.py - get_article - 27 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 17:58:51,416 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-03 18:02:39,648 - INFO - orm_get_article.py - get_article - 27 - relationship 'foreign_to_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:02:39,685 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: relationship 'foreign_to_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:02:44,263 - INFO - orm_get_article.py - get_article - 27 - relationship 'foreign_to_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:02:44,294 - INFO - orm_get_user_info.py - get_user_info - 32 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: relationship 'foreign_to_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:15:36,899 - INFO - orm_get_article.py - get_article - 27 - relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:15:36,951 - INFO - api_get_session.py - get_session - 30 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:16:27,771 - INFO - orm_get_article.py - get_article - 27 - relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:17:16,735 - INFO - orm_get_article.py - get_article - 27 - relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:17:33,841 - INFO - orm_get_article.py - get_article - 27 - relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:17:36,764 - INFO - orm_get_article.py - get_article - 27 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: relationship 'relate_comments' expects a class or a mapper argument (received: <class 'sqlalchemy.sql.schema.Table'>)
2020-07-03 18:19:07,619 - INFO - orm_get_article.py - get_article - 27 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-07 17:16:13,583 - INFO - orm_get_article.py - get_article - 27 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-07 17:37:07,414 - INFO - orm_login.py - login - 29 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-07 17:37:30,133 - INFO - orm_login.py - login - 29 - One or more mappers failed to initialize - can't proceed with initialization of other mappers. Triggering mapper: 'mapped class User->user'. Original exception was: When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-07 17:41:29,757 - INFO - orm_login.py - login - 29 - When initializing mapper mapped class User->user, expression 'Comments' failed to locate a name ('Comments'). If this is a class name, consider adding this relationship() to the <class 'config.config_orm_initial.initialize_orm.<locals>.User'> class after both dependent classes have been defined.
2020-07-07 18:03:47,576 - INFO - orm_get_user_info.py - get_user_info - 26 - name 'target_article' is not defined
2020-07-07 18:05:33,300 - INFO - orm_get_user_info.py - get_user_info - 26 - name 'target_article' is not defined
2020-07-07 22:33:57,091 - INFO - api_get_comments.py - get_comments - 23 - name 'json' is not defined
2020-07-07 22:34:32,930 - INFO - api_get_comments.py - get_comments - 23 - the JSON object must be str, bytes or bytearray, not NoneType
2020-07-07 22:36:00,574 - INFO - api_get_comments.py - get_comments - 24 - the JSON object must be str, bytes or bytearray, not NoneType
2020-07-07 22:37:46,362 - INFO - api_get_comments.py - get_comments - 24 - the JSON object must be str, bytes or bytearray, not NoneType
2020-07-07 22:38:26,415 - INFO - api_get_comments.py - get_comments - 24 - module 'orm.orm_get_comments' has no attribute 'get_article'
2020-07-07 22:38:54,564 - INFO - api_get_comments.py - get_comments - 24 - get_comments() missing 1 required positional argument: 'comments_range'
2020-07-07 22:57:14,963 - INFO - orm_get_comments.py - get_comments - 30 - 'Articlelist' object has no attribute 'relate_comments'
2020-07-08 17:06:32,701 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:06:41,317 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:42:59,432 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:44:06,198 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:47:40,286 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:48:14,476 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:50:17,893 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-08 17:52:21,388 - INFO - orm_get_user_info.py - get_user_info - 27 - 'NoneType' object has no attribute 'id'
2020-07-09 15:50:55,125 - INFO - api_save_cover.py - savecover - 22 - local variable 'current_article_path' referenced before assignment
2020-07-09 16:10:36,653 - INFO - api_save_cover.py - savecover - 47 - [Errno 2] No such file or directory: './static/articles/2/cover/cover.jpg'
